zebra/zebra_nhg.h

```diff
@@ -282,12 +282,17 @@ extern bool zebra_nhg_dependents_is_empty(const struct nhg_hash_entry *nhe);
 /* Lookup ID, doesn't create */
 extern struct nhg_hash_entry *zebra_nhg_lookup_id(uint32_t id);
 
+/* Lookup preserved, doesn't create */
+extern struct nhg_hash_entry *zebra_presvd_nhg_lookup(struct nhg_hash_entry *lookup);
```

Comment:
Could you name this function without shortening it with something like `presvd`? I don't see any reason to shorten and then guess what it means.